src/main/java/frc/lib/refvalues/RefValue.java

```diff
@@ -0,0 +1,17 @@
+package frc.lib.refvalues;
+
+public class RefValue<T> {
+    public T value;
```

Comment:
Why is it public if you have a public get?

Reply:
At first, it didn't have `get()` and `set()`, but because it was quite annoying that you needed to write `() -> value` and `newValue -> value = newValue` each time you wanted to use a supplier or consumer of this value, I added these methods. It seemed a bit weird to me too but I wasn't sure that I should remove the ability to use RefValue by just accessing value, as we regularly did, as it is also nice to have the ability to do things like `counter.value++`, do you think I should remove it anyway?

Reply:
hmmmm, it's pretty weird you can access it that way.
Will the user use it themselves?

Reply:
I mean, they can...? I can remove that and change it back if needed.